rules-tests/test.js

```diff
@@ -0,0 +1,165 @@
+const firebase = require('@firebase/testing');
+const fs = require('fs');
+
+/* eslint-env node, mocha */
+
+/*
+ * ============
+ *    Setup
+ * ============
+ */
+const projectId = 'cv19assist-firestore-testing';
+const firebasePort = require('../firebase.json').emulators.firestore.port;
+
+const port = firebasePort || 8080;
+const coverageUrl = `http://localhost:${port}/emulator/v1/projects/${projectId}:ruleCoverage.html`;
+
+const rules = fs.readFileSync('./firestore.rules', 'utf8');
+
+/**
+ * Creates a new app with authentication data matching the input.
+ *
+ * @param {object} auth the object to use for authentication (typically {uid: some-uid})
+ * @return {object} the app.
+ */
+function authedApp(auth) {
+  return firebase.initializeTestApp({ projectId, auth }).firestore();
+}
+function adminApp(auth) {
+  return firebase.initializeAdminApp({ projectId, auth }).firestore();
+}
+
+/*
+ * ============
+ *  Test Cases
+ * ============
+ */
+
+const TEST_UID = 'UID123';
+const TEST_UID2 = 'UID456';
+
+beforeEach(async () => {
```

Comment:
Given the placement of this block, is the intention that it clears the database for _each_ test within `describe`?  If so, I think that might not be what's happening here.  See https://mochajs.org/#hooks

Reply:
This is where Google's own QuickStart example had it, and I tested it with a console.log and it is firing each time.